app/Agri.Models/Configuration/Feed.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace Agri.Models.Configuration
+{
+    public class Feed : Versionable
+    {
+        public Feed()
+        {
+            FeedForageTypes = new List<FeedForageType>();
+        }
+
+        [Key]
+        public int Id { get; set; }
+
+        public string Name { get; set; }
+
+        [Column(TypeName = "decimal(16,4)")]
+        public decimal? CPPercent { get; set; }
+
+        [Column(TypeName = "decimal(16,4)")]
+        public decimal? PhosphorousPercent { get; set; }
+
+        [Column(TypeName = "decimal(16,4)")]
+        public decimal? PotassiumPercent { get; set; }
+
+        public List<FeedForageType> FeedForageTypes { get; set; }
```

Comment:
This is wrong. 

Feed is a Forage Type.

Change to:

        public FeedForageType GetFeedForageType { get; set; }